q2_vsearch/plugin_setup.py

```diff
@@ -404,12 +406,21 @@
                   'abundances).'),
     },
     parameter_descriptions={
+        'method': ('Which algorithm to use.'),
+        # 'abskew': ('The abundance skew is used to distinguish in a threeway '
+        #            'alignment which sequence is the chimera and which are '
+        #            'the parents. The parent sequences must be this many '
+        #            'times more abundant than the child sequence to be '
+        #            'flagged as chimeric.'),
         'dn': ('No vote pseudo-count, corresponding to the parameter n in '
                'the chimera scoring function.'),
-        'mindiffs': 'Minimum number of differences per segment.',
-        'mindiv': 'Minimum divergence from closest parent.',
+        'mindiffs': 'Minimum number of differences per segment. '
+                    'Ignored for uchime2 and uchime3.',
+        'mindiv': 'Minimum divergence from closest parent. '
+                  'Ignored for uchime2 and uchime3.',
```

Comment:
This is the best we can do without making different actions so don't worry about this.